internal/pkg/auth/sso/auth_sso_flow.go

```diff
@@ -45,26 +45,22 @@ func Login(authURL string, noBrowser bool, connectionName string) (string, strin
 		// we need to start a background HTTP server to support the authorization code flow with PKCE
 		// described at https://auth0.com/docs/flows/guides/auth-code-pkce/call-api-auth-code-pkce
 		server := newServer(state)
-		err = server.startServer()
-		if err != nil {
+		if err := server.startServer(); err != nil {
 			return "", "", errors.Wrap(err, errors.StartHTTPServerErrorMsg)
 		}
 
 		// Get authorization code for making subsequent token request
-		err := browser.OpenURL(state.getAuthorizationCodeUrl(connectionName, isOkta))
-		if err != nil {
+		if err := browser.OpenURL(state.getAuthorizationCodeUrl(connectionName, isOkta)); err != nil {
 			return "", "", errors.Wrap(err, errors.OpenWebBrowserErrorMsg)
 		}
 
-		err = server.awaitAuthorizationCode(30 * time.Second)
-		if err != nil {
+		if err = server.awaitAuthorizationCode(30 * time.Second); err != nil {
```

Comment:
Unrelated, but I'm thinking we want to remove the timer here, and let the user exit with Ctrl+C. The 30s timeout is arbitrary and sometimes it takes me longer than 30s to get my phone and accept the MFA request.